cluster-scope/overlays/ocp-prod/quota/disinformation-at-scale-eldritchjs/limitrange.yaml

```diff
@@ -0,0 +1,16 @@
+name: default
+spec:
+  limits:
+  - type: Container
+    default:
+      cpu: "4"
+      memory: "16Gi"
+      nvidia.com/gpu: 1
+    defaultRequest:
+      cpu: "4"
+      memory: "16Gi"
+      nvidia.com/gpu: 1
```

Comment:
@larsks would this mean that each container that's spawned will need to have a GPU? do we even need to have limitrange for this project?

Reply:
That's def not what I was going for (a GPU per container) so if I need to delete that line, no problem.

Reply:
@naved001 I changed the GPU requests to 0 per looking through the docs. The max is still 1, the default is now 0.